pkg/signature/kms/gcp/go.mod

```diff
@@ -0,0 +1,43 @@
+module github.com/sigstore/sigstore/pkg/signature/kms/gcp
+
+replace github.com/sigstore/sigstore => ../../../../
```

Comment:
I believe the replace directive breaks installation now, is it needed?

Reply:
I believe it can break `go install` _of binaries in this module_, but not those in subsequent downstream consumers of this module.

I might be wrong though, and we should try `go install`ing `cosign@main` et al depending on this commit to make sure, before we bump their deps.